src/prime_server.cpp

```diff
@@ -482,7 +479,6 @@ worker_t::worker_t(zmq::context_t& context,
   downstream_proxy.setsockopt(ZMQ_SNDHWM, &disabled, sizeof(disabled));
   downstream_proxy.connect(downstream_proxy_endpoint.c_str());
 
-  loopback.setsockopt(ZMQ_SNDHWM, &disabled, sizeof(disabled));
```

Comment:
we should revert this in keeping with the rest of the design of the project. its better to let the application max out ram than to arbitrarily limit to 1k or whatever the default is